### ISSUE TITLE
task - OTU radial - Asserted distribution - lock the source

### ISSUE THREAD

**Issue by proceps** [SpeciesFileGroup/taxonworks#1421]
There is already option to lock the source to quickly enter multiple areas for the same OTU/source. Is it possible to keep the same lock when OTU radial is closed and reopened (switching to another OTU for example)? Using cookies? That would be extremely helpful.

**Comment by proceps** [comment]
In addition, is it possible to preserve the same source selected when I switch to biological associations?

**Comment by mjy** [comment]
Is there a blue pin button beside all the forms you want?  We agreed that that was the mechansim in non-dedicated tasks rather than auto-population.

**Comment by proceps** [comment]
I used both and found that radial is way more convenient. I am staying on the same species page and can do everything in the same form rather than jumping from one task to another. Blue pin is great, but, the biggest problem is the page number, I have to type it every single time.

**Comment by mjy** [comment]
I'm not sure I like the idea of locks that persist across open and closing the radial annotator.  It's going to make people confused?

**Comment by proceps** [comment]
A check box persist or not?

**Comment by proceps** [comment]
The thing is, you know that you have 2 peaces of information to save the record. If one is already prepopulated. Not locked, just set in the right place. I do not see how this could be confusing. You can always delete the string and put another value.

**Comment by mjy** [comment]
It's potentially very confusing- you make one citation, you pull up another
record after some time, and a page is there- now I have to delete it
(another step).   This will be strongly dependent on the type of citation,
etc.  I don't see this working.

**Comment by proceps** [comment]
In most cases you are working on the same publication. I am working on a book with a thousand of species, and few thousand distribution records. At the same time, there are nomenclatural changes and host plant records. I am staying on the same page browse nomenclature for prolonged period of time. Even Session variable will work. If I come back after an hour, yes, the selection will be lost, but I can quickly restore. 
About saving a record unintentionally with wrong citation. Two days ago (still working on the same book). I always use blue pin button to return the citation. I temporary put another source as a default (Same authors, different year). And forgot to switch it back to the book. I saved about 50 records before I realized I am pulling wrong source from my pinboard. And I know the interface very well. Yes. Mistakes are always possible.
As an intermediate solution. remember only the page number. And if I pulled the source from the Pinboard, and this is the same source, prepopulate the pages.
For me, we always should think about productivity. For now to enter a single distribution record I have to:
1. Pull a citation from the pinboard (assuming it is already pinned and made default)
2. take the mouse click to the page text box
3. go back to keyboard and type the page number which could be 3-4 digits.
4. go back to mouse select original if it is the first record
5. go to the smart selector and switch to Serach.
6. back to the keyboard Search for the georgaphic area
7. back to mouse, select the geographic area from the list

And this is for a single record with two pieces of information (citation and geographic area).
